internal/policy/verify.go

```diff
@@ -686,7 +604,30 @@ func verifyEntry(ctx context.Context, repo *git.Repository, policy *State, attes
 	return nil
 }
 
-func verifyTagEntry(ctx context.Context, repo *git.Repository, policy *State, entry *rsl.ReferenceEntry) error {
+func verifyAttestation(ctx context.Context, verifiers []*Verifier, commitObj *object.Commit, authorizationAttestation *sslibdsse.Envelope) (bool, string, error) {
+	// No verifiers => no restrictions for the git namespace
+	if len(verifiers) == 0 {
```

Comment:
Let's handle this separately please, we want fewer "valid" return states. Currently, both `true, ""` and `true, verifierName` are valid. Let's make it so we only have the latter. This helper shouldn't be invoked if len(verifiers) == 0. This could also mean that we don't need a boolean return at all; we have an error returned if verification fails, or error==nil and the name of the verifier if successful.